install/install.go

```diff
@@ -14,6 +14,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+// Package install contains functionality to install the OpenSSF Scorecard workflow
```

Comment:
```suggestion
// Package install contains functionality to install the OpenSSF Scorecard workflow.
```

Linter should flag that package comments should have end with a period.